felt-armor-api/src/main/java/io/github/feltmc/feltapi/api/armor/ArmorTickItem.java

```diff
@@ -0,0 +1,13 @@
+package io.github.feltmc.feltapi.api.armor;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public interface ArmorTickItem {
+    /**
+     * Called to tick armor in the armor slot. Override to do something
+     */
+    default void onArmorTick(ItemStack stack, World level, PlayerEntity player) {
```

Comment:
why use defaults if its empty? and why are the brackets on separate lines if they are empty